src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -535,17 +602,51 @@ SELECT create_reference_table('table2_groupF');
 (1 row)
 
 -- check metadata
-SELECT * FROM pg_dist_colocation 
-    WHERE colocationid >= 1 AND colocationid < 1000 
+SELECT * FROM pg_dist_colocation
+    WHERE colocationid >= 1 AND colocationid < 1000
     ORDER BY colocationid;
- colocationid | shardcount | replicationfactor | distributioncolumntype 
```

Comment:
Both is acceptable for me. Apart from that my only concern would be seeing results of the queries as soon as possible. For example before "-- check failing colocate_with options" we create several distributed tables but there is no immediate output which shows the state of the tables, colocation groups etc.